.github/workflows/main.yml

```diff
@@ -10,16 +10,20 @@ on:
 
 jobs:
   build:
-    name: (build stencil ${{ matrix.stencil_version }})
```

Comment:
😆 oh I don't wanna know how long that's been there

Reply:
haha yeah looks like a copypasta thing